src/components/views/StackPage/Stack.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const Stack = (props) => {
+	const handleSelectStack = (e) => {
+		const isStack = e.target.textContent.split('.')[0];
+		// e.target.classList.toggle('select');
+		if (e.target.className === 'stack-chip') {
+			e.target.classList.add('select');
+		} else {
+			e.target.classList.remove('select');
+		}
```

Comment:
```suggestion
		e.target.classList[1] 
		    ? e.target.classList.remove('select') 
		    : e.target.classList.add('select');
```

위와 같은 조건도 좋을것 같아요!
className 자체도 좋지만 classList를 가져와서 안에 select가 있는지 없는지 확인하는것도 방법입니다!
이건 수정안해도 괜찮아요 :)

Reply:
넵! 이 방법으로도 해보겠습니다~!